README.markdown

```diff
@@ -347,3 +347,15 @@ If setting the `default` alias does not establish the node version in new shells
 [3]: https://travis-ci.org/creationix/nvm
 [Urchin]: https://github.com/scraperwiki/urchin
 [Fish]: http://fishshell.com
+
+## Uninstall
+
+```
+rm -rf $NVM_DIR ~/.npm ~/.bower && \
```

Comment:
In no way will `~/.npm` or `~/.bower` have any impact whatsoever on nvm's install script. I don't even want to mention `.bower` anywhere because nobody should ever be using bower, and I don't want to mention removing `~/.npm` because I don't want anybody destroying their npm cache.